packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -88,7 +88,7 @@ const LocaleModalCountries = ({
         <Search
           data-autoid={`${stablePrefix}--locale-modal__filter`}
           placeHolderText={modalLabels.searchPlaceholder}
-          labelText={modalLabels.searchLabel}
+          labelText={`${modalLabels.searchLabel}. Note: Location/Language will be dynamically updated as you provide the input`}
```

Comment:
We probably need to set this as something configurable, in case this is translated in the other 15 languages we support. We can make english the default value.

Reply:
Opened a [PR in ibm-dotcom-translations](https://github.ibm.com/webstandards/ibm-dotcom-translations/pull/251) to address this. At the moment I've only modified the `us-en`/default value for `searchLabel`, once approved the change will be reflected in the live versions.

For now, this current PR still reads out the number of results as intended.

cc: @Praveen-sr

Reply:
Thanks @IgnacioBecerra , I approved the translation change so that part should be good. Can you check the CI checks and see what's failing?